### PR TITLE
Create harvard-deakin-university.csl

### PR DESCRIPTION
**Pull request by simtyrell**
Minor changes have been made to reflect Deakin university harvard style referencing. 
changed the wording 'accessed' to 'retrieved' and removed the hanging indent.

### CODE REVIEW

File: harvard-deakin-university.csl [hunk 1]
```diff
@@ -0,0 +1,214 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Harvard - Deakin University </title>
+    <id>http://www.zotero.org/styles/harvard-deakin-university</id>
+    <link href="http://www.zotero.org/styles/harvard-deakin-university" rel="self"/>
```
Comment (adam3smith) [inline harvard-deakin-university.csl]: Is this style based on an existing style? If so, please add with rel="template" link. See https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link

File: harvard-deakin-university.csl [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+    <link href="http://www.deakin.edu.au/students/studying/study-support/referencing/harvard" rel="documentation"/>
+        <category citation-format="author-date"/>
+    <category field="generic-base"/>
+    <summary>Style for Deakin University </summary>
+    <updated>2016-09-15T11:36:24+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
```
Comment (adam3smith) [inline harvard-deakin-university.csl]: you can delete all of these terms except editor, translator, and edition short forms. They're already in the locale.

Comment (simtyrell) [inline harvard-deakin-university.csl]: Hi - I'm a little confused - I used this style as a template http://www.zotero.org/styles/harvard-the-university-of-melbourne. This one is already approved and in the style library, I only changed some wording and the hanging indent. All these other terms are in there and approved?  I a bit nervous to start deleting things as I'm not confident.

File: harvard-deakin-university.csl (hunk 1 above)
Comment (simtyrell) [inline harvard-deakin-university.csl]: I made this change and I got the email saying there was an issue with the style:

  1) harvard-deakin-university: must have a "self" link
     Failure/Error: expect(style).to have_self_link
       expected #has_self_link? to return true, got false
     Shared Example Group: "style" called from ./spec/styles_spec.rb:156
     # ./spec/styles_spec.rb:26:in `block (2 levels) in <top (required)>'
 134772/134772 |==================== 100 =====================>| Time: 00:00:55 
Finished in 55.31 seconds (files took 43.75 seconds to load)
134772 examples, 1 failure
Failed examples:
rspec ./spec/styles_spec.rb[1348:5] # harvard-deakin-university: must have a "self" link

File: harvard-deakin-university.csl [hunk 3]
```diff
@@ -0,0 +1,214 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Harvard - Deakin University </title>
+    <id>http://www.zotero.org/styles/harvard-deakin-university</id>
+    <link href="http://www.zotero.org/styles/harvard-deakin-university" rel="template"/>
```
Comment (adam3smith) [inline harvard-deakin-university.csl]: no, this should be

```
<link href="http://www.zotero.org/styles/harvard-deakin-university" rel="self"/>
<link href="http://www.zotero.org/styles/harvard-the-university-of-melbourne" rel="template"/>
```

File: harvard-deakin-university.csl (hunk 2 above)
Comment (adam3smith) [inline harvard-deakin-university.csl]: yeah, we've become more stringent on reviewing new style submissions. There's not much you can break here. Just delete all terms except the ones mentioned.

Comment (simtyrell) [inline harvard-deakin-university.csl]: ok - I think I have deleted the ones you needed done. thanks for your help.

File: harvard-deakin-university.csl (hunk 3 above)
Comment (simtyrell) [inline harvard-deakin-university.csl]: Thanks - I made this change - appreciate you help.

**Comment by adam3smith** [comment]
Great, thanks!

**Pull request merged**